packages/location_background/README.md

```diff
@@ -2,6 +2,8 @@
 
 An example Flutter plugin that showcases background execution using iOS location services.
 
+**This plugin is not being actively maintained. An [archive](https://github.com/flutter/plugins/tree/af86b9db204d4fee2d0eeabb2325007ec5f20424/packages/location_background) of previous versions can be found in the Flutter plugins repository.**
```

Comment:
Maybe say that it shouldn't be used in real apps and was only built as a sample?

Reply:
+1